src/services/annotation/service/auth.mjs

```diff
@@ -0,0 +1,27 @@
+import { fetch } from 'undici';
+
+import { authenticationEndpoint } from '../config.mjs';
+
+export const authenticate = async (email, token) => {
+	const body = JSON.stringify({ email, token });
+	const response = await fetch(
+		authenticationEndpoint,
+		{
+			body,
+			method: 'POST',
+			headers: { 'Content-Type': 'application/json' }
+		}
+	);
+	const result = await response.json();
+	return result;
+};
+
+export const parseBasicAuth = header => {
+	const base64String = header.slice(6, -1);
+	const buff = Buffer.from(base64String, 'base64');
+	const utfString = buff.toString('utf-8');
+	const [ email, token ] = utfString.split(':');
+	return { email, token };
+};
+
+// console.log(await authenticate('samuel.doogan@nesta.org.uk', '8cbbdc79d19e4cfb3235b42eb6651b93'));
```

Comment:
Possibly I would have this in a test

Reply:
woops - I'll have to generate a new auth token for myself, shouldn't have left this in the commit 🙄

Reply:
we agreed to postpone to avoid the complexity (e.g. you need to visit the link sent to your email in order to validate that you have a Nesta email) as we are now working on v1 of the annotation API